From 38a366a97b1a1c4e6d7eeb9ac140bf1f5ee9d68d Mon Sep 17 00:00:00 2001
From: Xu Jianqun <jay.xu@rock-chips.com>
Date: Wed, 6 Jan 2016 13:07:13 +0800
Subject: [PATCH] ARM64: configs: rockchip_defconfig select saradc

Select CONFIG_ROCKCHIP_SARADC of rockchip_defconfig for
Rockchip develop board.

Change-Id: Ic6e2f7632801222360528cbbaf62e3c59799cfd9
Signed-off-by: Xu Jianqun <jay.xu@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 539ba2a52537..d31134435587 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -443,6 +443,7 @@ CONFIG_RK_IOMMU=y
 CONFIG_RK_IOVMM=y
 CONFIG_PM_DEVFREQ=y
 CONFIG_IIO=y
+CONFIG_ROCKCHIP_SARADC=y
 CONFIG_PWM=y
 CONFIG_PWM_ROCKCHIP=y
 CONFIG_PHY_ROCKCHIP_USB=y
-- 
2.35.3

